specification/dimensions/usageunit.md

```diff
@@ -2,10 +2,20 @@
 
 Usage Unit refers to a unit of measurement for the consumption or usage of resources or services. The Usage Unit for a particular line item may differ from the published Pricing Unit when providers use different units, unit increments, or dimensions for calculating cost.  This unit is commonly used when auditing or reconciling consumption or billing data.
 
-The UsageUnit column MUST be present and MUST NOT be null or empty. UsageUnit is of type String, and UsageUnit MUST be composed of the list of recommended units or unit abbreviations if it is possible to be expressed with the units listed in "Recommended Unit Set" below. If the UsageUnit value is a composite value made from combinations of one or more units, each component MUST also align with the set of recommended units if possible.  For example, a composite unit of "GB/Month" is allowed.
+The UsageUnit column MUST be present and MUST NOT be null or empty. UsageUnit is of type String.
 
 Instead of "per" or "-" to denote a Composite Unit, slash ("/") SHOULD be used as a common convention.  Count based units like requests, instances, tokens SHOULD be expressed as "count".  For example, if a usage unit is measured as a rate of requests or instances over a period of time, the unit should be listed as "count/day" to signify the number of requests per day.
 
+## Mandatory Alignment with Recommended Unit Set
```

Comment:
Can we just merge this with the other section below?

Reply:
+1, I'll revise based on all this feedback, and then resubmit the PR against the feature branch.

Reply:
Merged